frontend/src/pages/org/collection-detail.ts

```diff
@@ -166,10 +166,14 @@ export class CollectionDetail extends BtrixElement {
                   <sl-button
                     size="small"
                     @click=${() => (this.openDialogName = "editStartPage")}
-                    ?disabled=${!this.collection?.crawlCount}
+                    ?disabled=${!this.collection?.crawlCount ||
+                    !this.isRwpLoaded}
                   >
-                    <sl-icon name="house-gear" slot="prefix"></sl-icon>
-                    ${msg("Configure Home")}
+                    ${!this.collection ||
+                    Boolean(this.collection.crawlCount && !this.isRwpLoaded)
+                      ? html`<sl-spinner slot="prefix"></sl-spinner>`
+                      : html`<sl-icon name="gear" slot="prefix"></sl-icon>`}
+                    ${msg("Configure View")}
```

Comment:
Updated this button text as well, since we no longer reference "home" (cc @Shrinks99 )

<img width="306" alt="Screenshot 2025-01-14 at 2 56 04 PM" src="https://github.com/user-attachments/assets/91b06bdb-2446-4f7d-8490-bf2a31083c1e" />

And in action menu:

<img width="175" alt="Screenshot 2025-01-14 at 2 55 56 PM" src="https://github.com/user-attachments/assets/7c2d492f-d8b2-4127-aadf-e9d0c9109ae6" />